core/src/main/java/jenkins/model/GlobalConfiguration.java

```diff
@@ -75,4 +76,13 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
         return Jenkins.get().getDescriptorList(GlobalConfiguration.class);
         // pointless type parameters help work around bugs in javac in earlier versions http://codepad.org/m1bbFRrH
     }
+
+    /**
+     * Assume by default that GlobalConfiguration items require Jenkins.ADMINISTER permission
+     * Ovveride to return something different if appropriate
+     * @return Permission
+     */
+    public Permission getPermission() {
+        return Jenkins.ADMINISTER;
+    }
```

Comment:
This logic should now be overridden in the Global Configure fields that should be accessible with CONFIGURE permission?

In previous commits for FGP, we added some logic to show/hide those options, Should we use this `getPermission` now to decide whether to show them or not, so this logic is centralized?

Reply:
thats something i wanted to discuss, i think this approach might be cleaner, at least from that perspective.